x-pack/plugins/reporting/public/panel_actions/get_csv_report_panel_action.tsx

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+
+import rison from 'rison-node';
+import {
+  ContainerState,
+  ContextMenuAction,
+  ContextMenuActionsRegistryProvider,
+  Embeddable,
+} from 'ui/embeddable';
+import { kfetch } from 'ui/kfetch';
+import { toastNotifications } from 'ui/notify';
+import { SearchEmbeddable } from '../../../../../src/legacy/core_plugins/kibana/public/discover/embeddable/search_embeddable';
```

Comment:
I'm not entirely sure this will work in a build, because of how x-pack gets relocated as a plugin

Reply:
yep, you are right, it won't.